core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -245,6 +245,16 @@ sealed trait ZValidation[+W, +E, +A] { self =>
       ZIO.succeedNow
     )
 
+  /**
+   * Returns the value, because no error has occurred.
+   */
+  final def value(implicit ev: E <:< Nothing): A = self.asInstanceOf[Success[W, A]].value
```

Comment:
`get` and `getOrElseWith`? Can probably just fold with `ev` here.

Reply:
I decided not to use `ev`, because I have `NonEmptyChunk[E]` and although I could go out of my way to "prove" that is "absurd", it would probably be more hassle than it's worth it.

Reply:
👍